demos/classical_ct_headneck_affine/README.md

```diff
@@ -1,17 +1,17 @@
-# Classical Affine Registration for Head-and-Neck CT Images
+# Classical affine registration for head-and-neck CT images
 
 > **Note**: Please read the
 > [DeepReg Demo Disclaimer](introduction.html#demo-disclaimer).
 
+[Source Code](https://github.com/DeepRegNet/DeepReg/tree/master/demos/classical_ct_headneck_affine)
+
 This is a special demo that uses the DeepReg package for classical affine image
 registration, which iteratively solves an optimisation problem. Gradient descent is used
 to minimise the image dissimilarity function of a given pair of moving anf fixed images.
 
 ## Author
 
-DeepReg Development Team (raise an issue:
-https://github.com/DeepRegNet/DeepReg/issues/new, or mailto the author:
-yipeng.hu@ucl.ac.uk)
+DeepReg Development Team
```

Comment:
maybe it's still better to put your name? @YipengHu 

later with more demos, the proportion of your name will decrease lol

Reply:
I will put my name then.... :D

Reply:
i would encourage to raise ticket instead of going to individuals.

Reply:
yes, but we just gave name without giving email.

if you want, you can merge the author & contact section lol

Reply:
maybe, after this PR